src/mrunal/maxNumber.java

```diff
@@ -0,0 +1,98 @@
+/*QUESTION: write a single program having different methods.
+ 
+a) To find Min number from given array
+b) To find Max number from given array
+c) To find Avg of all numbers from give array
+d) To display all numbers [0 to length]
+e) To display all numbers [length to 0]
+g) Method to find average of min and max number from array.*/
+
+package mrunal;
+
+public class maxNumber {
+	int index=0;
+	int getMinNumber(int[] input)
+	{
+		int min =input[0];
+		for(int index=0; index<input.length;index++)
+		{
+			if(input[index]<min)
+				min=input[index];
+		}
+		return min;
+	}
+	
+	
+	int getMaxNumber(int[] input)
+	{
+		int max =input[0];
+		for(int index=0; index<input.length;index++)
+		{
+			if(input[index]> max)
+				max=input[index];
+		}
+		return max;
+}
+	int getAverage(int[] input)
+	{
+		int sum=0;
+		for(int index=0; index<input.length; index++)
+		sum += input[index];
+		int average = sum/input.length;
+		return average;
+	}
+
+	void displayIncrementOrder(int[] input)
+	{
+		for(index=0; index<input.length;index++)
+		{
+			System.out.println(input[index]);
+		}
+		
+	}
+	
+	void displayDecrementOrder(int[] input)
+	{
+		for(index=(input.length-1) ;index>=0 ;index--)
+		{
+			System.out.println(input[index]);
+		}
+	}
+	
+	void averageOfMinMax(int[] input)
+	{
+		maxNumber m = new maxNumber();
+		
+		int ans1 = m.getMinNumber(input);
+		int ans = m.getMaxNumber(input);
+		int averageOfMinMax = (ans1 + ans)/2;
```

Comment:
You can write code like this, I remove variable and directly doing sum of min & max, no problem, if you are not comfortable to write code like below, just optimize.

int avgOfMinMax = (getMinNumber(input)+getMaxNumber(input))/2